stub_test/Makefile

```diff
@@ -6,30 +6,36 @@ CURDIR = $(shell pwd)
 
 GTEST_DIR = $(CURDIR)/googletest/googletest
 
+PROFILING_BINDINGS_FLAG=
+
 CC:= $(shell ocamlfind ocamlc -config | awk '/^bytecomp_c_compiler/ {for(i=2;i<=NF;i++) printf "%s " ,$$i}')
 LD=$(CC)
 
 all: $(BUILDDIR)/test
+
+testperf: PROFILING_BINDINGS_FLAG=-D PROFILE_BINDINGS
```

Comment:
For some reason when doing `make testperf` and then `make all`, it doesn't recompile the files accordingly (without the `PROFILING_BINDINGS_FLAG` set). I have to run `clean` in between. I've documented this in the README.